common/buildings/00_Castle.txt

```diff
@@ -13,582 +13,699 @@
+	}
+	ca_stable_3 = {
+		desc = ca_stable_1_desc
+		trigger = { TECH_CASTLE_CONSTRUCTION = 2 }
+		prerequisites = { ca_wall_2 }
+		upgrades_from = ca_stable_2
+		gold_cost = 100
+		build_time = 365
+		
+		# Warcraft
+		light_cavalry = 20
+		
+		ai_creation_factor = 87
+
+		extra_tech_building_start = 1.6
+	}
+	ca_stable_4 = {
+		desc = ca_stable_1_desc
+		trigger = { TECH_CASTLE_CONSTRUCTION = 4 }
+		upgrades_from = ca_stable_3
+		gold_cost = 150
+		build_time = 548
+		
+		# Warcraft
+		light_cavalry = 25
+		
+		ai_creation_factor = 86
+
+		extra_tech_building_start = 1.2
+	}
+	ca_stable_5 = {
+		desc = ca_stable_1_desc
+		trigger = { TECH_CASTLE_CONSTRUCTION = 6 }
+		upgrades_from = ca_stable_4
+		gold_cost = 200
+		build_time = 1095
+		
+		# Warcraft
+		light_cavalry = 30
+		
+		ai_creation_factor = 85
+
+		extra_tech_building_start = 0.8
+	}
+	ca_stable_6 = {
+		desc = ca_stable_1_desc
+		trigger = { TECH_CASTLE_CONSTRUCTION = 8 }
+		upgrades_from = ca_stable_5
+		gold_cost = 250
+		build_time = 1095
+		
+		# Warcraft
+		light_cavalry = 35
+		
+		ai_creation_factor = 85
+
+		extra_tech_building_start = 0.2
+	}
 	
-	# #Stable, gives light cavalry and some heavy cavalry
-	# ca_stable_1 = {
-		# desc = ca_stable_1_desc
-		# trigger = { TECH_CASTLE_CONSTRUCTION = 0 }
-		# gold_cost = 100
-		# build_time = 365
-		# light_cavalry = 35
-		# ai_creation_factor = 89
-		
-		# extra_tech_building_start = 0.8
-	# }
-	# ca_stable_2 = {
-		# desc = ca_stable_1_desc
-		# trigger = { TECH_CASTLE_CONSTRUCTION = 0 }
-		# prerequisites = { ca_wall_1 }
-		# upgrades_from = ca_stable_1
-		# gold_cost = 120
-		# build_time = 547
-		# light_cavalry = 45
-		# ai_creation_factor = 88
-
-		# extra_tech_building_start = 2.2
-	# }
-	# ca_stable_3 = {
-		# desc = ca_stable_1_desc
-		# trigger = { TECH_CASTLE_CONSTRUCTION = 2 }
-		# prerequisites = { ca_wall_2 }
-		# upgrades_from = ca_stable_2
-		# gold_cost = 200
-		# build_time = 730
-		# light_cavalry = 55
-		# ai_creation_factor = 87
-
-		# extra_tech_building_start = 1.6
-	# }
-	# ca_stable_4 = {
-		# desc = ca_stable_1_desc
-		# trigger = { TECH_CASTLE_CONSTRUCTION = 4 }
-		# upgrades_from = ca_stable_3
-		# gold_cost = 300
-		# build_time = 1095
-		# light_cavalry = 55
-		# knights = 5
-		# ai_creation_factor = 86
-
-		# extra_tech_building_start = 1.2
-	# }
-	# ca_stable_5 = {
-		# desc = ca_stable_1_desc
-		# trigger = { TECH_CASTLE_CONSTRUCTION = 6 }
-		# upgrades_from = ca_stable_4
-		# gold_cost = 400
-		# build_time = 2190
-		# light_cavalry = 55
-		# knights = 10
-		# ai_creation_factor = 85
-
-		# extra_tech_building_start = 0.8
-	# }
-	# ca_stable_6 = {
-		# desc = ca_stable_1_desc
-		# trigger = { TECH_CASTLE_CONSTRUCTION = 8 }
-		# upgrades_from = ca_stable_5
-		# gold_cost = 500
-		# build_time = 2190
-		# light_cavalry = 55
-		# knights = 20
-		# ai_creation_factor = 85
-
-		# extra_tech_building_start = 0.2
-	# }
+	# Warcraft
+	#Aviary, gives heavy cavalry AKA Aerial Units
```

Comment:
Mb create a new file for non-vanilla castle buildings that were added in MO?

Reply:
Meh, it will move them away from the stables. I believe they should be near each others since it's the same unit group.